bin/dune_init.ml

```diff
@@ -519,7 +519,43 @@ module Component = struct
             [ File.make_text (Path.parent_exn opam_file) (Path.basename opam_file) "" ]
           | Esy -> [ File.make_text dir "package.json" "" ]
         in
-        { dir; files = dune_project_file dir opts :: package_files }
+        let default_files =
+          let fmt_version =
+            try
+              let ic = Unix.open_process_in "ocamlformat --version" in
```

Comment:
Instead of `open_prcess_in`, I think it would be better to use Dune's `Process`.